src/Cement.cs

```diff
@@ -260,14 +259,14 @@ public void RevertEventSystem()
             _usingCementEventSystem = false;
         }
 
-        public void _Log(object o, BepInEx.Logging.LogLevel logLevel=BepInEx.Logging.LogLevel.Info)
+        private void InternalLog(object o, BepInEx.Logging.LogLevel logLevel=BepInEx.Logging.LogLevel.Info)
```

Comment:
This code won't work. You can't have a private void, because then in Log, it won't be able to access InternalLog from Singleton

Reply:
holy crap i was wrong